tests/customport/docker-compose.yml

```
@@ -0,0 +1,9 @@
+version: '3.7'
+
+services:
+  customport-test:
+    image: customport-check-agent
+    build: ../src/connectivity
+    network_mode: host
+    environment:
+      AGENT-PORT: 5001
```

A dash in env vars is a big no no
```suggestion
      AGENT_PORT: 5001
```